python_modules/dagster-externals/dagster_externals_tests/test_external_execution.py

```diff
@@ -101,7 +111,11 @@ def foo(context: AssetExecutionContext, ext: SubprocessExecutionResource):
                 extras=extras,
                 context_injector=context_injector,
                 message_reader=message_reader,
-                env={"CONTEXT_INJECTOR_SPEC": context_injector_spec},
+                env={
+                    "CONTEXT_INJECTOR_SPEC": context_injector_spec,
+                    "MESSAGE_READER_SPEC": message_reader_spec,
+                    **aws_env_vars,
```

Comment:
This turns out to be harder than anticipated because we are communicating cross-process via S3. From what I can tell the moto mock S3 state is process-scoped. Do you know if moto has a way to share its state across processes.